savant/metrics/base.py

```diff
@@ -1,7 +1,5 @@
 from abc import ABC, abstractmethod
 
-from savant.metrics.metric import Metric
-
 
 class BaseMetricsExporter(ABC):
```

Comment:
Do you think we still need BaseMetricExporter?

Reply:
I will check it!

Reply:
@denisvmedyantsev removed.